deploy-manage/deploy/cloud-enterprise/ece-manage-capacity.md

```diff
@@ -31,17 +31,22 @@ ECE does not support hot-adding of resources to a running node.  When increasing
 ::::
 
 
-To adjust the allocator capacity, prior to ECE 3.5.0; it is necessary to reinstall ECE on the host with a new value assigned to the `--capacity` parameter. From ECE 3.5.0 onwards, just use the ECE API :
+To adjust the allocator capacity prior to ECE 3.5.0, you must reinstall ECE on the host with a new value assigned to the `--capacity` parameter. Starting with ECE 3.5.0, you can update the allocator capacity using the ECE API. After making this change, you must restart the allocator service for it to take effect.
```

Comment:
maybe this needs a sub-heading to separate the planning vs. adjusting task

suggestion: consider also pulling the pre-3.5.0 info into a note below the opening paragraph. reason being that a customer might decide to skip this paragraph if they're on 3.6+ . you could even merge the 3.5.0 note below up into a single note

also, you could share a link to the ECE API docs if you felt like it on "ECE API" :)

Reply:
great point, thanks

Reply:
https://www.elastic.co/docs/api/doc/cloud-enterprise/operation/operation-set-allocator-settings